packages/api/amplify_api/test/dart_graphql_test.dart

```diff
@@ -147,7 +147,11 @@ class MockAmplifyAPI extends AmplifyAPIDart {
 
   @override
   WebSocketConnection getWebSocketConnection({String? apiName}) =>
-      MockWebSocketConnection(testApiKeyConfig, getTestAuthProviderRepo());
+      MockWebSocketConnection(
+        testApiKeyConfig,
```

Comment:
got some failures here, thinking it might help to share the util for generating mock ws connection.